Python/hbayesdm/base.py

```diff
@@ -423,16 +428,68 @@ def _prepare_pars(self, model_regressor: bool, inc_postpred: bool) -> List:
             pars += self.postpreds
         return pars
 
+    def _prepare_gen_init_vb(self,
+                             data_dict: Dict,
+                             n_subj: int,
+                             ) -> Union[str, Callable]:
+        """Prepare initial values for the parameters using Variational Bayesian
+        methods.
+
+        Parameters
+        ----------
+        data_dict
+            Dict holding the data to pass to Stan.
+        n_subj
+            Total number of subjects in data.
+
+        Returns
+        -------
+        gen_init : Union[str, Callable]
+            A function that returns initial values for each parameter, based on
+            the variational Bayesian method.
+        """
+        model = self._get_model_full_name()
+        sm = self._designate_stan_model(model)
+
+        try:
+            fit = sm.vb(data=data_dict)
+        except Exception:
+            raise RuntimeError(
+                'Failed to get VB estimates for initial values. '
+                'Please re-run the code to try fitting model with VB.')
```

Comment:
@JaeyeongYang 
Are there any plans to also fix the base.py code part for running hbayesdm with vb=True ? (Regarding #92)